pages/index.js

```diff
@@ -1,7 +1,9 @@
+import Feed from "./components/feed";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <Feed />
     </div>
```

Comment:
Did you know that you could use [Fragments](https://react.dev/reference/react/Fragment#returning-multiple-elements) to return multiple elements instead of a div?